dev/Repeater/VirtualizationInfo.h

```diff
@@ -48,6 +48,9 @@ class VirtualizationInfo : public winrt::implements<VirtualizationInfo, winrt::I
     winrt::IInspectable Data() const { return m_data.get(); }
     winrt::IDataTemplateComponent DataTemplateComponent() const { return m_dataTemplateComponent.get(); }
 
+    bool MustClearDataContext() { return m_mustClearDataContext; }
```

Comment:
const to keep it consistent with the other methods in this class.

Reply:
Fixed